dash/types/src/pairql/pairs/UserActivitySummaries.ts

```diff
@@ -1,20 +1,15 @@
 // auto-generated, do not edit
 
 export namespace UserActivitySummaries {
-  export interface Input {
-    userId: UUID;
-    dateRanges: Array<{
-      start: string;
-      end: string;
-    }>;
-  }
```

Comment:
the summaries pair no longer takes a set of date ranges. we never did anything but always pass in a hardcoded "most recent 14 days" anyway, and now things are different because we want 14 days of normal activity, and anything older that is flagged (flagged items don't auto delete for 60 days). so i made that baked into the pair, instead of requiring an array of date ranges